aerogear/mobile-services-installer
# feature: add support for rhcc

The change will allow us to support `rhcc` registry, and it will be the registry that will have the productized images.

To verify:

Run `ansible-playbook install-mobile-services.yml -e "ansible_playbookbundle_registry_type=rhcc"` and check the configuration of the ansible service broker. There should be a new registry added that's pointing to rhcc.
